### PR TITLE
Assign tasks 1

### PR DESCRIPTION
**Pull request by isaac342368**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "@react-native-async-storage/async-storage": "^1.18.2",
     "@react-native-community/clipboard": "^1.5.1",
     "@react-native-community/datetimepicker": "^7.2.0",
+    "@react-native-picker/picker": "2.4.10",
```
Comment (aarontorres0) [inline package.json]: This package already exists in main so no need to add it. Can also remove changes in package-lock.json.

File: app/screens/HomeScreen.jsx [hunk 2]
```diff
@@ -70,7 +70,8 @@ const HomeScreen = () => {
         name: taskName,
         body: taskBody,
         dueDate: date,
-        // TODO: add a createdBy value
+        createdBy: username,
+        assignedTo: assignedTo// TODO: add a createdBy value
```
Comment (aarontorres0) [inline app/screens/HomeScreen.jsx]: remove comment

**Review by aarontorres0 — CHANGES_REQUESTED** [review]

**Review by jlipscomb071 — APPROVED** [review]
LGTM

**Review by aarontorres0 — APPROVED** [review]
lgtm

**Pull request merged**